src/views/manage/ManageFormStyle.jsx

```diff
@@ -0,0 +1,24 @@
+import styled from '@datapunt/asc-core';
```

Comment:
stop deze in de file waar hij gebruikt wordt

Reply:
This is my preference for this project, should increase readability